packages/react-native-renderer/src/ReactNativeFiberInspector.js

```diff
@@ -190,13 +190,21 @@ if (__DEV__) {
 
           closestInstance =
             internalInstanceHandle.stateNode.canonical._internalInstanceHandle;
+
+          // Note: this is deprecated and we want to remove it ASAP. Keeping it here for React DevTools compatibility for now.
+          const nativeViewTag = internalInstanceHandle.stateNode.canonical._nativeTag;
+
           nativeFabricUIManager.measure(
             internalInstanceHandle.stateNode.node,
             (x, y, width, height, pageX, pageY) => {
+              const inspectorData = getInspectorDataForInstance(
+                getClosestInstanceFromNode(closestInstance),
```

Comment:
What's this needed for?

Reply:
the change is just a refactor so this codepath looks like the non-Fabric path